internal/oonirun/experiment_test.go

```diff
@@ -70,10 +67,12 @@ func TestExperimentRunWithFailureToSubmitAndShuffle(t *testing.T) {
 					MockNewTargetLoader: func(config *model.ExperimentTargetLoaderConfig) model.ExperimentTargetLoader {
 						return &mocks.ExperimentTargetLoader{
 							MockLoad: func(ctx context.Context) ([]model.ExperimentTarget, error) {
-								// Implementation note: the convention for input-less experiments is that
-								// they require a single entry containing an empty input.
-								entry := model.NewOOAPIURLInfoWithDefaultCategoryAndCountry("")
-								return []model.ExperimentTarget{entry}, nil
+								results := []model.ExperimentTarget{
+									model.NewOOAPIURLInfoWithDefaultCategoryAndCountry("a"),
+									model.NewOOAPIURLInfoWithDefaultCategoryAndCountry("b"),
+									model.NewOOAPIURLInfoWithDefaultCategoryAndCountry("c"),
+								}
+								return results, nil
```

Comment:
This fixes a test that I broke previously. Now that we have a loader, we need to return "a", "b", "c" here rather than using the Inputs fields above. With this change we're back at 100% coverage.